moztelemetry/histogram.py

```diff
@@ -8,15 +8,25 @@
 from __future__ import division
 
 import requests
-import parse_histograms
 import re
 import pandas as pd
 import numpy as np
 import ujson as json
 
-from functools32 import lru_cache
+from . import parse_histograms
 from expiringdict import ExpiringDict
 
+# python 2 and 3 compatiblity
+try:
+    from functools32 import lru_cache
+except ImportError:
+    from functools import lru_cache
+
+try:
```

Comment:
This should use `from past.builtins import long` (and keep code as-is) or `from builtins import int` (and change the `long` to `int`)